src/browser/modules/Sidebar/UserSettings.tsx

```diff
@@ -159,6 +159,13 @@ const visualSettings = [
             'If this is checked, after a cypher query result is retrieved, a second query is executed to fetch relationships between result nodes.',
           type: 'checkbox'
         }
+      },
+      {
+        showWheelZoomInfo: {
```

Comment:
Personally I don't think it makes sense to show this setting to the user, I can't imagine myself ever wanting to see it again, but that's more up to product